test/ruby/test_iseq.rb

```diff
@@ -125,6 +125,16 @@ def test_lambda_with_ractor_roundtrip
     assert_equal(42, ISeq.load_from_binary(iseq.to_binary).eval)
   end
 
+  def test_super_with_anonymous_block
+    iseq = compile(<<~EOF)
+      def touch3(&block) # :nodoc:
```

Comment:
I think you meant `def touch3(&) # :nodoc:` here, no?

Reply:
Yep, I'll fix that.

Reply:
Submitted a PR for it: #5147